### DISCUSSION TITLE
User Account remove group, show "Invalid credentials" when login

### DISCUSSION THREAD

**Discussion by SimonChiu930** [rejetto/hfs#786] [Q&A] [answered]
I create the account group "test" and add the account "peter" inherit from group, permission has been OK.
I remove "test" group on "peter" account, show the error "Invalid credentials" when login.
How to solve this issue?

![WeChat Screenshot_20241023161018](https://github.com/user-attachments/assets/b44565a4-b80d-4de1-a620-0282088bb63c)

**Comment by rejetto** [comment]
are you trying to to log in as "peter" ?
i did what you said, and i'm not getting the error.
can you copy me your config?
admin-panel > options > config file > copy
and then paste here or send to a@rejetto.com

**Reply by SimonChiu930** [reply]
Video Capture sent to your email address. thanks

**Comment by rejetto** [comment, marked as answer]
problem solved by update 0.53.2
for users of 0.54: it's already fixed there

**Reply by SimonChiu930** [reply]
I updated to 0.53.2, this problem was solved.
Thank your support
